orga/translations/en.json

```diff
@@ -571,6 +571,17 @@
         }
       },
       "personalised-test-title": "Title of the customised test",
+      "reset-to-zero": {
+        "title": "Reset to zero",
```

Comment:
non-blocking

Pour le prescrit on dit ["Reset and restart from the beginning"](https://github.com/1024pix/pix/blob/483f637385c3261dc39c02f14da1ea3d252ac7bd/mon-pix/translations/en.json#L1440), peut être faut-il les nommer pareil ? @PhilippineLoison

Reply:
Oui - c'est un bon remarque